### ISSUE TITLE
Which devices / cameras actually support this?

### ISSUE THREAD

**Issue by Regaddi** [w3c/mediacapture-image#307]
Hey 👋 

I kind of accidentally stumbled across the Image Capture API and I'm amazed by the amount of controls that are technically available.
However I wonder if there's somewhere a list of devices / cameras that actually support this API.
I've heard of people connecting their DSLR cameras via HDMI (e.g. using the Elgato Camlink 4K), but in those cases the interesting controls (like ISO, color temperature, pan/tilt/zoom, etc.) are not available, because the video signal is already processed on the device and sent through the HDMI out.
Connecting DSLR cameras via USB seems very unreliable, given that most of the utility software provided by the camera vendors aren't that good.

Do you happen to have a list of devices at hand that fully support the Image Capture API? Is there something like an "Open Source DSLR"?

Thanks a lot in advance!

Christian

**Comment by fippo** [comment]
I don't think this is the right place to ask but maybe @beaufortfrancois knows anyway

**Comment by beaufortfrancois** [comment]
https://github.com/w3c/mediacapture-image/blob/main/implementation-status.md should help. There's no list of devices though as every device exposes/supports different settings/capabilities.
